Expensify/App
# Enable haptic feedback on the NumberPad

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/7128

### Tests | QA Steps
1. Open App on Android and IOS **devices**.
2. Click FAB (:heavy_plus_sign: ).
3. Open request money.
4. Add some value via the number pad. 
5. All keys should have haptic/vibration feedback on each key press and on long press.


### Tested On


### Screenshots


#### Web

![image](https://user-images.githubusercontent.com/24370807/154808458-897aed6d-51bc-48a2-ac09-ba2b6057041d.png)

#### Mobile Web

![image](https://user-images.githubusercontent.com/24370807/154808568-49dc6e7d-8291-457b-b0e4-27355ca9bfe8.png)

#### Desktop

![image](https://user-images.githubusercontent.com/24370807/154808464-0c7835a2-e9fb-4e85-b251-9b29bf4ed6eb.png)

#### iOS


#### Android

![image](https://user-images.githubusercontent.com/24370807/154808447-dafb327c-54c3-4338-92e9-001d194c58e9.png)
